alchemy_cms.gemspec

```
@@ -22,7 +22,7 @@ Gem::Specification.new do |gem|
   gem.add_runtime_dependency 'acts_as_list',                     ['>= 0.3', '< 2']
   gem.add_runtime_dependency 'awesome_nested_set',               ['~> 3.1']
   gem.add_runtime_dependency 'cancancan',                        ['>= 2.1', '< 4.0']
-  gem.add_runtime_dependency 'coffee-rails',                     ['~> 4.0', '< 5.0']
+  gem.add_runtime_dependency 'coffee-rails',                     ['>= 4.0', '<= 6.0']
```

As @mamhoff already mentioned we would like to be conservative here and stay below the next major version.
```suggestion
  gem.add_runtime_dependency 'coffee-rails',                     ['>= 4.0', '< 6.0']
```

Would you mind changing that?